openshift-environment-driver/src/main/java/org/jboss/pnc/environment/openshift/Configurations.java

```diff
@@ -28,30 +28,49 @@
 /**
  * @author <a href="mailto:matejonnet@gmail.com">Matej Lazar</a>
  */
-public enum Configurations {
+public class Configurations {
```

Comment:
I changed it to class to change the intent of the 'class' from being fairly static (enum) to more dynamic (with property name to specify template file). I can change it back to enum though and add the logic in `getContentsAsString()`